addons/website_slides_forum/views/slide_channel_views.xml

```diff
@@ -25,9 +25,6 @@
         <field name="model">slide.channel</field>
         <field name="inherit_id" ref="website_slides.slide_channel_view_form_add"/>
         <field name="arch" type="xml">
-            <xpath expr="//group[@name='review']" position="attributes">
-                <attribute name="string">Communication</attribute>
-            </xpath>
```

Comment:
You received specs about form view changes or did you choose? (could be ok but if you choose by yourself, maybe worth validating with PDE) :+1:

Reply:
Confirmed with PDE & LNA and added some UI tweaks (requested by LNA), so this is the final form :+1: 

![master-new-course-preview-mou](https://user-images.githubusercontent.com/25032876/187256848-3290efad-2a99-4221-b166-bf5f3b9c9584.png)